src/http/routes.cc

```diff
@@ -140,6 +140,11 @@ routes& routes::add(operation_type type, const url& url,
     return add(rule, type);
 }
 
+routes& routes::add_default_handler(handler_base* handler) {
```

Comment:
nitpick: I think `set_default_handler` would be a better name. `add` suggests that you can add more than 1, while only a single instance is stored. Also, removing a default handler would be done with `add_default_handler(nullptr)`, which looks a little weird.

Reply:
still, not a blocker at all, I'll commit the change. Thanks for the contribution!